Apps/Sandcastle/templates/bucket-requirejs.html

```diff
@@ -8,10 +8,12 @@
     <script type="text/javascript" src="../Sandcastle-header.js"></script>
     <script type="text/javascript" src="../../../ThirdParty/requirejs-2.1.20/require.js"></script>
     <script type="text/javascript">
-    require.config({
-        baseUrl : '../../../Source',
-        waitSeconds : 60
-    });
+        if(typeof require === "function") {
+            require.config({
+                baseUrl : '../../../Source',
+                waitSeconds : 60
```

Comment:
Can we bump this to 120 or something?  I know this PR is about making this faster not slower, but sometimes (especially loading legacy releases off the site) this timeout is too short.

Reply:
I can make this a little longer, but this change will only affect future release builds.

Reply:
Actually, the the require call shouldn't happen in the built version regardless, so it will only slow down the dev version.

Reply:
Yeah this was kind of a tangential comment, I know you didn't set this value in this PR.